### ISSUE TITLE
Delete page without warning

### ISSUE THREAD

**Issue by muguu** [jaap-karssenberg/zim-desktop-wiki#361]
Please confirm if "Delete page" shows a warning prior to deletion.

**Comment by Osndok** [comment]
For me, there is no confirmation upon requesting a page to be deleted.

**Comment by jaap-karssenberg** [comment]
It depends on whether a thrash is supported or not. If delete is permanent, it will ask for confirmation. If thrashing it will not but you can restore from your system trash.

On windows trashing will show a popup as well because the operating system asks for confirmation.

Would like to have the trash-undo directly in zim, but afraid that is not platform independent.

**Comment by myrdd** [comment]
tl;dr this is a comment as well as ”kind of“ a bug report

Thank you for moving deleted content to the trash instead of removing permanently! :+1: 

I delete pages *accidentally* quite often, because I use the keyboard to execute commands. In en-US locale, `Alt+F` opens the `File` menu, and the `D` accesskey is assigned `Delete Page`. (So I just need to accidentally hit `D` when the File menu is open, and everything is deleted (*recursively!*).)

**Please note** that recovering deleted pages (and subpages) from trash **does not always recover all data**: When a page is deleted, any link to that page or any of its subpages is being ”removed“ (converted to text-only) **except** the ”*Remove links when deleting pages*“ setting (Preferences—Interface) is being unchecked. IMHO when that option is *checked* and links are about to be removed, there *should* always be a confirmation dialog. What do you think @jaap-karssenberg?

For myself, I work around the issue by simply keeping the ”*Remove links when deleting pages*“ setting unchecked.

**Comment by mysterymeth** [comment]
@jaap-karssenberg Is there any way we can modify this in the settings, I would really like confirmation AND trashing, sometimes my mouse control isn't great!

**Comment by arigit** [comment]
@jaap-karssenberg can we please have the option to display a simple OK/Cancel cancellation when deleting a page, regardless of whether trash is supported?

As @myrdd commented above, it's easy to delete pages without noticing. Further, when using the Index in a HiDPI monitor, it's easy to select the wrong page one wants to delete. In those cases, having the content in the Trash does not help - if one doesn't realize content was lost, eventually that content will be unknowingly purged from the trash. This is kind of important to avoid losing content and applies to many common use cases. E.g.

If you agree this is worth addressing, I'd suggest reopening this bug for  tracking

As of Zim 0.72 the no-confirmation behavior is still present.
